examples/version.tf

```diff
@@ -0,0 +1,16 @@
+##############################################################################
+# Terraform Providers
+##############################################################################
+
+terraform {
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = "~>1.41.0"
```

Comment:
Lock into lowest version in example to ensure we don't end up with invalid lower range. So that would be `1.41.0`